packages/utils/scss/_mixins.scss

```diff
@@ -26,3 +26,9 @@
         }
     }
 }
+
+@mixin generate-utils-multiple( $map, $values, $function: "", $important: $kendo-important) {
+    @each $side in $map {
+        @include generate-utils( $side, $side, $values, $function: "", $important: $kendo-important );
+    }
+}
```

Comment:
As discussed in our offline meeting, please drop this mixin and define mappings in the `_variables` file. Then, use the standard `generate-utils()` mixin to generate the classes.